### PR TITLE
Add development guide

### PR DESCRIPTION
**Pull request by jeffersonking**

### CODE REVIEW

File: README.develop.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Development Guide
```
Comment (EasyRhinoMSFT) [inline README.develop.md:1]: The standard filename we are using for this is CONTRIBUTING.md

Comment (jeffersonking) [reply README.develop.md:1]: Looks like `CONTRIBUTING.md` already exists (last edit 4 years ago!). I will starting rationalizing and merging the two.

File: README.develop.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Overview
+This document guides starting development on the SARIF Viewer extension for Visual Studio Code.
```
Comment (VCOLAWTON) [inline README.develop.md:4]: Rewrite: The instructions in this document will help you get started with the SARIF Viewer extension for Visual Studio Code. @jeffersonking

**Review by VCOLAWTON — CHANGES_REQUESTED** [review]
Please make the change to the overview and submit to team for peer review. Thanks, Jeff!

File: README.develop.md (hunk 2 above)
Comment (jeffersonking) [reply README.develop.md]: Updated.

File: README.develop.md (hunk 1 above)
Comment (jeffersonking) [reply README.develop.md]: Merged.

**Review by EasyRhinoMSFT — APPROVED** [review]
:shipit:

**Pull request merged**